fedimint-server/Cargo.toml

```diff
@@ -20,12 +20,14 @@ bitcoin = "0.29.2"
 bitcoin_hashes = "0.11.0"
 bytes = "1.4.0"
 hbbft = { git = "https://github.com/jkitman/hbbft", branch = "upgrade-threshold-crypto-libs" }
+fedimint-api = { path = "../fedimint-api" }
+fedimint-core = { path = "../fedimint-core" }
+fedimint-wallet = { path = "../modules/fedimint-wallet", features = ["native"] }
 futures = "0.3.24"
 impl-tools = "0.6.1"
 itertools = "0.10.5"
 jsonrpsee = { version = "0.16.2", features = ["server"] }
-fedimint-api = { path = "../fedimint-api" }
-fedimint-core = { path = "../fedimint-core" }
+mint-client = { path = "../client/client-lib" }
```

Comment:
Why does the server rely on the mint client? Maybe that's a bit too involved and should go into a separate PR, but would be good to know. (pinging @jkitman who also worked on modularization)

Reply:
We just finished a refactor to avoid the `fedimint-server` depending on modules, we should keep it that way.